vsproject/mrustc.vcxproj

```diff
@@ -168,6 +168,7 @@
       <AdditionalDependencies>common_lib.lib;mrustc_lib.lib;%(AdditionalDependencies)</AdditionalDependencies>
       <AdditionalLibraryDirectories>$(OutDir)</AdditionalLibraryDirectories>
       <AdditionalOptions>/WHOLEARCHIVE:mrustc_lib.lib %(AdditionalOptions)</AdditionalOptions>
+      <StackReserveSize>8388608</StackReserveSize>

```

Comment:
Out of curiosity, what part needed the larger stack?

Reply:
IIRC it was while doing constant evaluation while compiling rustc (or one of its dependencies).

Reply:
I should also note that I only tested the full bootstrap when combined with my other PR to update to VS2019. I don't have the VS2015 compiler installed anymore. I expect it should work, but it's possible there may be some differences from the new compiler.